contracts/Orders.sol

```diff
@@ -260,73 +260,12 @@ contract Orders is Withdrawable, Utils2 {
         );
     }
 
-    function getNextOrder(uint32 orderId) public view returns(uint32, bool isLast) {
+    function getNextOrder(uint32 orderId)
+        public
+        view
+        returns(uint32, bool isLast)
+    {
```

Comment:
I prefer the one line syntax if possible
Don't u?

Reply:
Following the Solidity style guide, if it exceeds the line length we should break it.